.github/workflows/release.yml

```diff
@@ -0,0 +1,194 @@
+name: release
+on:
+  release:
+    types: [published]
+    tags:
+      - v*
+  # Uncomment this if you'd like to test the build+release process on PRs too
+  # (it will only test the build, it wont push to PyPI until a new release is tagged)
+  # push:
+  #   branches:
+  #     - master
+  # pull_request:
+  #   branches:
+  #     - master
+
+env:
+  LIBZIM_RELEASE: libzim_linux-x86_64-6.1.1
+  LIBZIM_LIBRARY_PATH: lib/x86_64-linux-gnu/libzim.so.6.1.1
+  LIBZIM_INCLUDE_PATH: include/zim
+  CYTHON_VERSION: 0.29.6
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        # TODO: expand this to cross-platform builds (see V2 approach below)
+        # os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+
+      - name: Cache libzim dylib
+        uses: actions/cache@master
+        id: cache-libzim
+        with:
+          path: libzim_linux
+          key: $LIBZIM_RELEASE-libzim-cache
+
+      - name:  Install libzim dylib & headers in system
+        if: steps.cache-libzim.outputs.cache-hit != 'true'
+        run: |
+          wget -q https://download.openzim.org/release/libzim/$LIBZIM_RELEASE.tar.gz
+          tar --gunzip --extract --file=$LIBZIM_RELEASE.tar.gz
+          mv $LIBZIM_RELEASE libzim_linux
+
+      - name: Sync libzim dylib & headers in system
+        run: |
+          sudo rsync -a $GITHUB_WORKSPACE/libzim_linux/$LIBZIM_LIBRARY_PATH /usr/lib/libzim.so
+          sudo rsync -a $GITHUB_WORKSPACE/libzim_linux/$LIBZIM_INCLUDE_PATH/ /usr/include/zim/
```

Comment:
It would be easier to set the CPPFLAS to `-I$GITHUB_WORKSPACE/libzim_linux/$LIBZIM_INCLUDE_PATH -L$GITHUB_WORKSPACE/libzim_linux/$LIBZIM_LIBRARY_PATH` instead of copying file in the (root) fs

Reply:
We have to `mv` it anyway because `cython` can't find the `libzim.so` file via the `libzim.so.6.1.1` symlink for some reason, it only works when it's named `libzim.so`.

Installing it system-wide also makes it more convenient later as python finds it automatically during the `setup.py` Cython and `cythonize` process.

Reply:
this is now fixed to use relataive `./lib` and `./include` in the github workspace dir

Reply:
> because cython can't find the libzim.so file via the libzim.so.6.1.1 symlink for some reason, it only works when it's named libzim.so

It would be good why. "Packaged installation" doesn't provide a libzim.so but only a versioned library (`so.6.1.1`, `so.6`, ...).
The plain unversionned `.so` is used a link time. (It is why it is in the `-dev` packages).

Reply:
I don't know any more detail unfortunately, other than the fact that if you set LD_LIBRARY_PATH to a folder that only contains the `.so.6.1.1` and `so.6` it doesn't work, but if it contains the `.so` then it works.